recipes/pysbol/run_test.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Instead of adding a separate file, you should add the `pytest -v` command to the recipe itself:
```
test:
  ...
  commands:
    pytest -v
  ...
```
It’s easier to maintain that way.

Reply:
Hi @marcelm , I deleted the separate file and add `pytest -v` to the recipe but the tests failed with this error:

```
10:53:08 BIOCONDA INFO (ERR) [Dec  8 10:53:08] SERR bash: pytest: command not found
10:53:08 BIOCONDA INFO (ERR) [Dec  8 10:53:08] ERRO Task processing failed: Unexpected exit code [127] of container [a110877139b2 step-a2306b4b49], container preserved
.10:53:41 BIOCONDA INFO (OUT) ..
10:53:41 BIOCONDA ERROR COMMAND FAILED (exited with 1): mulled-build build-and-test pysbol=0.0.2--pyhdfd78af_0 -n biocontainers --test bash -c '/usr/local/env-execute true && . /usr/local/env-activate.sh && pytest -v' --channels conda-forge,file:///home/circleci/project/miniconda/conda-bld,bioconda,defaults --involucro-path /home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/involucro

10:53:41 BIOCONDA ERROR TEST FAILED: recipes/pysbol
10:53:41 BIOCONDA INFO (COMMAND) conda build purge
10:53:42 BIOCONDA ERROR BUILD SUMMARY: of 1 recipes, 1 failed and 0 were skipped. Details of recipes and environments follow.
10:53:42 BIOCONDA ERROR BUILD SUMMARY: FAILED recipe recipes/pysbol


Exited with code exit status 1

CircleCI received exit code 1
```

Reply:
Sorry about that, you need to move it back to the script because the 'test: requires:' dependency is not installed during the `mulled-build` test, see https://bioconda.github.io/contributor/guidelines.html#test-dependencies